Mathlib/Topology/Separation.lean

```diff
@@ -171,99 +171,99 @@ end SeparatedNhds
 /-- A T₀ space, also known as a Kolmogorov space, is a topological space such that for every pair
 `x ≠ y`, there is an open set containing one but not the other. We formulate the definition in terms
 of the `Inseparable` relation.  -/
-class T0Space (α : Type u) [TopologicalSpace α] : Prop where
+class T0Space (X : Type u) [TopologicalSpace X] : Prop where
   /-- Two inseparable points in a T₀ space are equal. -/
-  t0 : ∀ ⦃x y : α⦄, Inseparable x y → x = y
+  t0 : ∀ ⦃x y : X⦄, Inseparable x y → x = y
 #align t0_space T0Space
 
-theorem t0Space_iff_inseparable (α : Type u) [TopologicalSpace α] :
-    T0Space α ↔ ∀ x y : α, Inseparable x y → x = y :=
+theorem t0Space_iff_inseparable (X : Type u) [TopologicalSpace X] :
+    T0Space X ↔ ∀ x y : X, Inseparable x y → x = y :=
   ⟨fun ⟨h⟩ => h, fun h => ⟨h⟩⟩
 #align t0_space_iff_inseparable t0Space_iff_inseparable
 
-theorem t0Space_iff_not_inseparable (α : Type u) [TopologicalSpace α] :
-    T0Space α ↔ ∀ x y : α, x ≠ y → ¬Inseparable x y := by
+theorem t0Space_iff_not_inseparable (X : Type u) [TopologicalSpace X] :
+    T0Space X ↔ ∀ x y : X, x ≠ y → ¬Inseparable x y := by
```

Comment:
The downside of this new naming is that `x y : X` breaks symmetry in a way that `x y : α` does not; are there any lemmas in this file with both `y : X` and `z : Y` or similar?

Reply:
I checked and fixed all cases. (All cases of y : X are in the context `x y : X`. While this is asymetric, these names are common in math literature; may I call scope creep on *this*?)